subprojects/scala/src/integTest/groovy/org/gradle/integtests/fixtures/ZincCoverage.groovy

```diff
@@ -20,5 +20,5 @@ import static org.gradle.api.plugins.scala.ScalaBasePlugin.DEFAULT_ZINC_VERSION
 
 
 class ZincCoverage {
-    public static final List<String> ALL_VERSIONS = [DEFAULT_ZINC_VERSION, "1.6.0"]
+    public static final List<String> ALL_VERSIONS = [DEFAULT_ZINC_VERSION, "1.6.1", "1.6.0"]
```

Comment:
I did not add 1.7.0 to the list of versions since there should be no reason for anyone to use that, but please let me know if you want the versions listed here to be exhaustive.